stackrox/overlays/moc/smaug/set-server.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: stackrox-central-deployment
+  namespace: stackrox
+spec:
+  destination:
+    server: "smaug"
```

Comment:
can we try `jerry` first?

Reply:
I believe we can? I have never heard of the Jerry cluster before. Also I don't know if the server argument that I put down (smaug) is correct. I was going to wait for @HumairAK to look at it and review it with me.